perfkitbenchmarker/scripts/database_scripts/launch_mysql_service.py

```diff
@@ -198,18 +278,18 @@ def _execute_pkb_cmd(pkb_cmd, stdout_filename, stderr_filename):
   # Will probably have to implement with threading.
   p.wait()
   elapsed_time = time.time() - start_time
+  if elapsed_time == 1:
```

Comment:
why do you use the duration of the execution to infer the call has failed? Can we look at the status directly? Or look at the output/stderr file?